examples/vcr_training/global_vcr/global_vcr_training.ipynb

```diff
@@ -0,0 +1,364 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "eaaa7c44-6469-44c4-b1bd-e1a3d2a05e7b",
+   "metadata": {},
+   "source": [
+    "## 1. Settings"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ad108e2-45c0-4baf-91de-3685c67586b0",
+   "metadata": {},
+   "source": [
+    "## 1.1 Setting up the environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8aa3e74b-eea5-46a2-8d1b-77e5b2bb6202",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "from typing import Any, List\n",
+    "import os\n",
+    "sys.path.append(\"../../../../../../tbmalt\")\n",
```

Comment:
It would be best to remove the hardcoded module path definition, `sys.path.append("../../../../../../tbmalt")`, from the import header. As this is only valid in the situation where TBMaLT has been downloaded but not installed and its location relative to the documentation/example package exactly matches that specified.